### PR TITLE
add Spotify team members to techdocs-maintainers and organization

### PR DESCRIPTION
**Pull request by agentbellnorm**
## Hey, I just made a Pull Request!

Adding engineers from the Spotify team that owns TechDocs to techdocs-maintainers and organization to enable them to contribute.

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: OWNERS.md [hunk 1]
```diff
@@ -139,6 +145,7 @@ Scope: Tooling for frontend and backend schema-first OpenAPI development.
 | Patrick Jungermann             | Bonial International GmbH | [pjungermann](https://github.com/pjungermann)         | `pjungermann#6933`             |
 | Phil Kuang                     | FactSet Research Systems  | [kuangp](https://github.com/kuangp)                   | `pkuang#3202`                  |
 | Taras Mankovski                | Frontside                 | [taras](https://github.com/taras)                     | `tarasm#1256`                  |
+| Vojta Burian                   | Spotify                   | [sankaritan](https://github.com/sankaritan)           | `vojtabu`                      |
```
Comment (Rugvip) [inline OWNERS.md:148]: These would go through a different process, see https://github.com/backstage/backstage/blob/master/GOVERNANCE.md#becoming-an-organization-member

File: OWNERS.md [hunk 2]
```diff
@@ -90,10 +90,15 @@ Team: @backstage/techdocs-maintainers
...
-| Raghunandan Balachandran | Spotify      | Pulp Fiction | [soapraj](http://github.com/soapraj)             | raghunandanb#1114  |
+| Name                     | Organization | Team    | GitHub                                            | Discord             |
+| ------------------------ | ------------ | ------- | ------------------------------------------------- | ------------------- |
+| Alex Lorenzi             | Spotify      | ProTean | [AlexLorenzi](https://github.com/AlexLorenzi)     | `alexlorenzi`       |
+| Bond Yan                 | Spotify      | ProTean | [byan1197](https://github.com/byan1197)           | `bondify`           |
+| Jackson Chen             | Spotify      | ProTean | [PeaWarrior](https://github.com/PeaWarrior)       | `jacksonc#3322`     |
+| John Philip              | Spotify      | ProTean | [johnphilip283](https://github.com/johnphilip283) | -                   |
+| Morgan Bentell           | Spotify      | ProTean | [agentbellnorm](http://github.com/agentbellnorm)  | `morganbentell`     |
+| Raghunandan Balachandran | Spotify      | BUX     | [soapraj](http://github.com/soapraj)              | `raghunandanb#1114` |
```
Comment (Rugvip) [inline OWNERS.md:100]: Just noting that it's fine to have Raghu here as a long-term maintainer of TechDocs, even if he's not on the owning team anymore.

**Review by Rugvip — CHANGES_REQUESTED** [review]

File: OWNERS.md (hunk 1 above)
Comment (agentbellnorm) [reply OWNERS.md]: Ah ok, I've removed Vojta and Daniel from the organization table, keeping only the project area maintainers in this PR.

**Review by PeaWarrior — APPROVED** [review]

**Review by awanlin — APPROVED** [review]
Thanks for this @agentbellnorm, this will really help us know who to ping on various TechDocs related PRs and Issues 🚀

**Review by Rugvip — APPROVED** [review]
👍

**Review by squid-ney — APPROVED** [review]

File: OWNERS.md [hunk 3]
```diff
@@ -90,10 +90,15 @@ Team: @backstage/techdocs-maintainers
...
-| Morgan Bentell           | Spotify      | Pulp Fiction | [agentbellnorm](http://github.com/agentbellnorm) | morganbentell#9030 |
-| Raghunandan Balachandran | Spotify      | Pulp Fiction | [soapraj](http://github.com/soapraj)             | raghunandanb#1114  |
+| Name                     | Organization | Team    | GitHub                                            | Discord             |
+| ------------------------ | ------------ | ------- | ------------------------------------------------- | ------------------- |
+| Alex Lorenzi             | Spotify      | ProTean | [AlexLorenzi](https://github.com/AlexLorenzi)     | `alexlorenzi`       |
+| Bond Yan                 | Spotify      | ProTean | [byan1197](https://github.com/byan1197)           | `bondify`           |
+| Jackson Chen             | Spotify      | ProTean | [PeaWarrior](https://github.com/PeaWarrior)       | `jacksonc#3322`     |
+| John Philip              | Spotify      | ProTean | [johnphilip283](https://github.com/johnphilip283) | -                   |
+| Morgan Bentell           | Spotify      | ProTean | [agentbellnorm](http://github.com/agentbellnorm)  | `morganbentell`     |
```
Comment (johnphilip283) [inline OWNERS.md:99]:
```suggestion
| John Philip              | Spotify      | ProTean | [johnphilip283](https://github.com/johnphilip283) | `john_philip#2399`                |
| Morgan Bentell           | Spotify      | ProTean | [agentbellnorm](http://github.com/agentbellnorm)  | `morganbentell`     |
```

**Review by byan1197 — APPROVED** [review]

**Pull request merged**